src/app.js

```diff
@@ -42,129 +45,131 @@ class MonotoneShape extends HTMLElement {
   }
 
   getSVGByShapeId(shapeId) {
+      const width=this.getAttribute("width");
+      const height=this.getAttribute("height");
     const svgs = {
-      1: `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      1: `<svg  width="${width?width:24}" height="${height?height:24}" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
 <rect x="9.125" y="7.95834" width="11.875" height="11.0833" rx="1" fill="#2C2C2D" fill-opacity="0.82"/>
 <rect x="2" y="4" width="12.3913" height="10.2917" rx="1" fill="#292929" fill-opacity="0.52"/>
 </svg>
 `,
-      2: `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      2: `<svg width="${width?width:24}" height="${height?height:24}" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
 <rect x="7" y="7" width="12" height="12" rx="6" fill="#2C2C2D" fill-opacity="0.82"/>
 <rect x="3" y="3" width="4" height="4" rx="2" fill="#292929" fill-opacity="0.52"/>
 </svg>
 `,
-      3: `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
now this can be made cleaner by simply writing the `width` without any ternary operator as we are using a null coalescence. 
Please format your code too.